src/components/plot/add/PlotAdd.tsx

```diff
@@ -36,7 +36,7 @@ export default function PlotAdd() {
   const fingerprint = useSelector(
     (state: RootState) => state.wallet_state.selected_fingerprint,
   );
-  const addNFTref = useRef();
+  const addNFTref = createRef();
```

Comment:
functional components are not using createRef. correct hook use useRef

Reply:
Sorry

Reply:
Fixing